pkg/util/utils.go

```diff
@@ -107,9 +109,10 @@ func RetryAfter(attempts int, callback func() error, d time.Duration) (err error
 			glog.Infof("non-retriable error: %v", err)
 			return m.ToError()
 		}
-		glog.V(2).Infof("sleeping %s", d)
+		glog.V(1).Infof("retriable error, sleeping %s", d)
 		time.Sleep(d)
 	}
+	glog.Infof("Returning: %v", m.ToError())
```

Comment:
Do we really need this? It seems it will most of the times log "Returning: MultiError{}" or similar